backend/src/apiserver/resource/resource_manager_util_test.go

```diff
@@ -263,7 +263,8 @@ spec:
   arguments: {}
   entrypoint: rand-fail-dag
   templates:
-  - dag:
+  - arguments: {}
```

Comment:
Hmm. I thought this Argo issue was fixed at some point...
Update: Turns out, not fully https://github.com/argoproj/argo/issues/1686